pkg/controller/controller.go

```diff
@@ -543,6 +547,26 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 		}
 	}
 
+	if len(cont.config.ApicHosts) != 0 && cont.vmmClusterFaultSupported {
+		//Clear fault instances when the controller starts
+		cont.clearFaultInstances()
+		//Subscribe for vmmEpPD for a given domain
+		var tnTargetFilterEpg string
+		tnTargetFilterEpg += fmt.Sprintf("uni/vmmp-Kubernetes/dom-%s", cont.config.AciVmmDomain)
```

Comment:
vmmp-Kubernetes can't be hardcoded. The vendor needs to come from config as it could be OpenShift instead